.github/workflows/ci.yml

```diff
@@ -1,6 +1,17 @@
 name: ci
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - "master"
+  pull_request:
+
+# Cancel previous PR builds.
+concurrency:
```

Comment:
Can this concurrency setting be used across all GHA workflows? Looks very nice.

Reply:
Yes. See https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#concurrency